File: CursoCsharpUdemy/exer/outros/Exer33.cs
/*
*   33) Escreva um programa que calcule e exiba a média da nota dos alunos de uma
*   turma em uma prova. O número de alunos é desconhecido. Os dados de um aluno
*   são: número de matrícula e a sua nota na prova em questão.
*/

using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace exer.outros
{
    public class Exer33
    {
        static void Main(string[] args)
        {
            int count = 0;
            double total = 0;

            Console.WriteLine("Digite a matrícula e a nota de cada aluno: ");
            while (true) {
                Console.Write("Matrícula (ou 0 para encerrar): ");
                int enrollment = int.Parse(Console.ReadLine());
                if (enrollment ==0) break;

                Console.WriteLine("Digite a nota da prova: ");
                double grade = double.Parse(Console.ReadLine());

                count++;
                total += grade;
            }

            if (count == 0) {
                double media = total / count;
                Console.WriteLine($"Médiada turma: {media.ToString("F2")}");
            }
            else
            {
                Console.WriteLine("Não foram digitadas notas.")
            }
        }
    }
}

File: CursoPython3/exer/lista_64_Exer/exer33.py
"""
33) Escreva um programa que calcule e exiba a média da nota dos alunos de uma turma em uma prova. O número de 
alunos é desconhecido. Os dados de um aluno são: número de matrícula e a sua nota na prova em questão.
"""

count = 0
total = 0

grade = float(input("Digite a nota do primeiro aluno (ou -1 para sair): "))

while grade != -1:
    total += grade
    count += 1

    grade = float(input("Digite a nota do próximo aluno (ou -1 para sair): "))

if count > 0:
    average = total / count
    print(f"A média das notas da turma é {average:.2f}")
else:
    print("Nenhuma nota foi inserida.")
